eng/Versions.props

```diff
@@ -30,7 +30,7 @@
     <RoslynDiagnosticsNugetPackageVersion>3.3.3-beta1.21105.3</RoslynDiagnosticsNugetPackageVersion>
     <MicrosoftCodeAnalysisNetAnalyzersVersion>6.0.0-preview1.21054.10</MicrosoftCodeAnalysisNetAnalyzersVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20623.3</MicrosoftCodeAnalysisTestingVersion>
-    <CodeStyleAnalyzerVersion>3.8.0</CodeStyleAnalyzerVersion>
+    <CodeStyleAnalyzerVersion>3.9.0</CodeStyleAnalyzerVersion>
```

Comment:
Interesting, I would have expected at least some new violations on command line with adopting a new CodeStyle package. Is CodeStyle enforcement broken on build? I’ll play around with it today.

Reply:
No clue :)  i took it to mean that our code is just that awesome.